blueprints/route/index.js

```diff
@@ -0,0 +1,147 @@
+/* eslint-env node */
+
+var fs                   = require('fs-extra');
```

Comment:
Should not use `var` anywhere in this file per https://github.com/softlayer/ember-style-guide/blob/master/javascript.md#variables